viadot/sources/genesys.py

```diff
@@ -152,33 +135,38 @@ def authorization_token(self, verbose: bool = False):
 
         return request_headers
 
-    def genesys_generate_exports(
-        self, post_data_list: List[str], end_point: str = "reporting/exports"
+    def genesys_api_connection(
+        self,
+        post_data_list: List[str],
+        end_point: str = "analytics/reporting/exports",
+        params: Dict[str, Any] = None,
+        method: str = "POST",
```

Comment:
Add information on what options are possible for this argument. `POST` and `GET`?